ui/packages/app/src/project_setting/metadata/MetadataList.js

```diff
@@ -0,0 +1,190 @@
+  const onLabelChange = labels => {
+    const labelsValid =
+      labels.length === 0
+        ? true
+        : labels.reduce((labelsValid, label) => {
+            return labelsValid && label.isKeyValid && label.isValueValid;
+          }, true);
+    setIsValidLabels(labelsValid);
+    if (!labelsValid) {
+      setLabelError(
+        "Invalid labels. Both key and value of a label must contain only alphanumeric and dash (-)"
+      );
+    }
+
+    if (
+      selectedTeam === project.team &&
+      selectedStream === project.stream &&
+      selectedLabels !== project.labels
+    ) {
+      setIsValidTeam(true);
+      setIsValidStream(true);
+      setIsValidLabels(labelsValid);
+      setTeam(selectedTeam);
+      setStream(selectedStream);
+    }
+
+    //deep copy
+    let newLabels = JSON.parse(JSON.stringify(labels));
+    newLabels = newLabels.map(element => {
+      delete element.isKeyValid;
+      delete element.isValueValid;
+      delete element.idx;
+      return element;
+    });
+
+    setLabels(newLabels);
+  };
+  return (
+    <>
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem>
+          <EuiTitle size="s">
+            <h3>Stream</h3>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          <EuiText size="s" color="subdued">
+            <p>Product stream the project belongs to</p>
+          </EuiText>
+          <EuiSpacer size="m" />
+          <EuiFormRow isInvalid={!isValidStream} error={streamError}>
+            <EuiComboBoxSelect
+              value={selectedStream}
+              options={streamOptions}
+              selectedOptions={selectedStream}
+              onChange={onStreamChange}
+              onCreateOption={onStreamChange}
+              isDisabled={!config.METADATA_UPDATE_ENABLED}
+            />
+          </EuiFormRow>
+          <EuiSpacer size="xl" />
+          <EuiTitle size="s">
+            <h3>Team</h3>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          <EuiText size="s" color="subdued">
+            <p>The owner of the project</p>
+          </EuiText>
+          <EuiSpacer size="m" />
+          <EuiFormRow isInvalid={!isValidTeam} error={teamError}>
+            <EuiComboBoxSelect
+              value={selectedTeam}
+              selectedOptions={selectedTeam}
+              options={teamOptions}
+              onChange={onTeamChange}
+              onCreateOption={onTeamChange}
+              isDisabled={!config.METADATA_UPDATE_ENABLED}
+            />
+          </EuiFormRow>
+          <EuiSpacer size="xl" />
+          <EuiTitle size="s">
+            <h3>Labels</h3>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          <EuiText size="s" color="subdued">
+            <p>Additional Labels</p>
+          </EuiText>
+          <EuiSpacer size="m" />
+          <EuiFormRow isInvalid={!isValidLabels} error={labelError}>
+            <Labels onChange={onLabelChange} initialLabels={project.labels} />
```

Comment:
can this be fixed? there is a lot of space but label values are being squeeze. 
<img width="664" alt="Screenshot 2024-05-07 at 12 20 49 PM" src="https://github.com/caraml-dev/mlp/assets/30390872/aa84b34a-01bb-431a-9d1c-dead17480876">

You can inspect and check the element, the sizing of the labels is shrinked and not fitting to the parent element